composer.json

```diff
@@ -13,10 +13,10 @@
         }
     ],
     "require":{
-        "php": ">=5.4",
-        "nikic/php-parser": "^2.0",
-        "symfony/console": "2.5 - 3.2",
-        "symfony/event-dispatcher": "2.4 - 3.4"
+        "php": ">=7.1",
+        "nikic/php-parser": "^4.0",
+        "symfony/console": "^4.1",
```

Comment:
why not leave compatibility with older Symfony versions?

Reply:
@CountZero1981 because the code can't be compatible with old versions and new versions at the same time. A choice has to be made. Stay in the past, or move forward to PHP 7.1 like a majority of the other PHP libraries/apps.

Reply:
but I'm not talking about php version. we're running our projects on php 7.1 / symfony 3.4 (lts so it's still maintained). I don't mind to set `php >=7.1`, but setting `symfony/console ^4.1` will make it incompatible with older Symfony versions without a reason.

Reply:
also, it would be nice to have phar archives with releases because then it can be installed as .phar where dependencies don't allow to install it as a composer package.

Reply:
@CountZero1981 there is no need to install it within a different application, outside as a separate project works fine and makes sure you don't suffer from dependency conflicts.
```
/build/your-app
/tools/psecio
```
works absolutely fine.